.github/workflows/ci.yml

```diff
@@ -0,0 +1,13 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches: main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - run: "echo 'TODO: tests'"
```

Comment:
no `stack test`?

Reply:
This project isn't real, so I didn't feel like doing a blind (via Code Suggestion) update that I'd probably get wrong, then have to check out and run and figure out... I just wanted it to not fail.